### ISSUE TITLE
Covenant Carbine or how it's too cool to use: Ammo count (Also Give ODSTs BRs)

### ISSUE THREAD

**Issue by Sgtwok** [HaloSpaceStation/HaloSpaceStation13#2457]
The weapons closest to the Covenant's carbine would be the DMR, and the BR.

When it comes to ammunition count, or even weapon count, the covenant is severely lacking, especially for kig-yar.

As it stands, kig-yar get 6 Carbines, and 24 magazines. That is 5 magazines (counting the one already loaded) if one doesn't take more than their share.

The sangheili have more ammo, but less carbines, at 3 carbines but 10 magazines.

The UNSC, per ODST vendor: Posses 2 DMRs with 16 magazines each, of which there are four vendors.
This puts ODSTs at 8 DMRs, along with 48 magazines altogether. Which means 8 magazines per weapon. That's a lot of focused dakka.
Though the don't have BRs, which I believe they should

For the regular UNSC Weapon vendors: 
4 Machines
5 DMRs per machine (times 4 equals 20)
25 DMR magazines (times 4 equals 100)
Per weapon: This is 5 Magazines 

This puts UNSC at 28 DMRs altogether, along with 148 magazines.

As for Battle Rifles:
2 Per Vendor (8)
20 per vendor (80)

This would give each battle rifle about 10 magazines per weapon.

In short: And much more convenient 
8 BRs with 80 Magazines 
28 (8 ODST) DMRs with 148 Magazines (48 ODST)
9 Carbines (3 Sangheili) with 78 Magazines (30 Sangheili)

Did I mention that UNSC can produce more ammo with the ammo foundry easily located near one of their dropship Landing zones?

I'm proposing an increase on carbine ammo count, perhaps even an increase on carbine rifle count given the low supply of merely 9 compared to the UNSC's 28 DMRs and 8 BRs (though they should be using more BRs anyway, since they are marines and not Army, this message is for you Cael <333333333)

This would be a temporary solution since there are potential proposals for an ammo resupply requisition system in the works.
